src/shared_modules/utils/rocksDBWrapper.hpp

```diff
@@ -41,6 +41,56 @@ namespace Utils
             m_db.reset(dbRawPtr);
         }
 
+        /**
+         * @brief Copy constructor.
+         *
+         * @param other Other instance.
+         */
+        RocksDBWrapper(RocksDBWrapper& other)
+            : m_db {std::move(other.m_db)}
+        {
+        }
+
+        /**
+         * @brief Move constructor.
+         *
+         * @param other Other instance.
+         */
+        RocksDBWrapper(RocksDBWrapper&& other) noexcept
+            : m_db {std::move(other.m_db)}
+        {
+        }
+
+        /**
+         * @brief Copy assignment operator.
+         *
+         * @param other Other instance.
+         * @return RocksDBWrapper&
+         */
+        RocksDBWrapper& operator=(RocksDBWrapper& other)
```

Comment:
The same comment that was made for the copy constructor

Reply:
Please, see the answer above